scripts/json_schema/gen_json_schema.py

```diff
@@ -18,6 +20,8 @@
 from marshmallow.class_registry import get_class
 from marshmallow_jsonschema import JSONSchema
 
+class Flow(YamlFileSchema):
```

Comment:
what's this for?

Reply:
let me remove this. I had an idea of create a Flow schema for the union of Flow and EagerFlow, and I finally give up.